forta-uniswap-event-py/README.md

```diff
@@ -0,0 +1,27 @@
+# Large Uniswap Detector Agent
+
+## Description
+
+This agent detects large uniswap transactions on ETH liquidity pools
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- AE-UNISWAP-LARGESWAP-EVENT
+  - Fired when a swap occurs where the value being traded is over 5 ether
+  - Only triggers when a Deposit or Withdrawal event occurs on the WETH contract via the Uniswap V2
+    Router contract
+
+## Test Data
```

Comment:
When only one handler is present in the README.md, it makes sense to have the `Test Data` section at the top level.  Once there are additional handlers (i.e. for a suite of handlers), I'm not certain whether the `Test Data` section should still be at the top level or if specific test data for each handler should fall under the handler listing in the `Alerts` section.

Reply:
If I understand you correctly, you're saying that it makes sense as it currently is because there is only one handler. We don't plan to make this into a suite of tool's so I'm going to resolve